### PR TITLE
Allow redeployment of assignments

### PR DESCRIPTION
**Pull request by celestemartinez**
- Allow redeployment of reviewer assignments if the Review stage hasn't started
- Allow redeployment of AC assignments if the MetaReview stage hasn't started

### CODE REVIEW

**Comment by mohituniyal** [comment]
Seems there are some merge conflicts that git is having troubles with.

File: tests/test_matching.py [hunk 1]
```diff
@@ -41,8 +41,8 @@ def test_setup_matching(self, client, test_client, helpers):
             </p>'''
         })
         print ('Homepage header set')
-        builder.set_conference_area_chairs_name('Senior_Program_Committee')
```
Comment (melisabok) [inline tests/test_matching.py:44]: is there a reason why you changed the group names?

File: tests/test_matching.py [hunk 2]
```diff
@@ -744,65 +634,207 @@ def test_setup_matching_with_subject_areas(self, client, test_client, helpers):
...
+        revs_paper0 = client.get_group(conference.get_id()+'/Paper{x}/Reviewers'.format(x=blinded_notes[0].number))
+        assert 2 == len(revs_paper0.members)
+
+        revs_paper1 = client.get_group(conference.get_id()+'/Paper{x}/Reviewers'.format(x=blinded_notes[1].number))
+        assert 2 == len(revs_paper1.members)
+
+        revs_paper2 = client.get_group(conference.get_id()+'/Paper{x}/Reviewers'.format(x=blinded_notes[2].number))
+        assert 2 == len(revs_paper2.members)
+       
+    def test_reploy_assigments(self, conference, client, test_client, helpers):
```
Comment (melisabok) [inline tests/test_matching.py:789]: there is typo here

File: tests/test_matching.py (hunk 1 above)
Comment (celestemartinez) [reply tests/test_matching.py:44]: We have to check which is the match group so we can delete the appropriate groups when redeploying, I am assuming now the groups are always 'Area Chairs' and 'Reviewers'

Comment (celestemartinez) [reply tests/test_matching.py:44]: I guess I could just use is_area_chair instead

**Comment by melisabok** [comment]
@celestemartinez there is a conflict when I try to update the branch with master, can you take a look? I this is ready to merge, thanks.

**Pull request merged**